tests/actions/ReportTest.js

```diff
@@ -30,21 +28,7 @@ jest.mock('../../src/libs/actions/Report', () => {
 
 describe('actions/Report', () => {
     beforeAll(() => {
-        // When using the Pusher mock the act of calling Pusher.isSubscribed will create a
-        // channel already in a subscribed state. These methods are normally used to prevent
-        // duplicated subscriptions, but we don't need them for this test so forcing them to
-        // return false will make the testing less complex.
-        Pusher.isSubscribed = jest.fn().mockReturnValue(false);
-        Pusher.isAlreadySubscribing = jest.fn().mockReturnValue(false);
-
-        // Connect to Pusher
-        PusherConnectionManager.init();
-        Pusher.init({
-            appKey: CONFIG.PUSHER.APP_KEY,
-            cluster: CONFIG.PUSHER.CLUSTER,
-            authEndpoint: `${CONFIG.EXPENSIFY.DEFAULT_API_ROOT}api?command=AuthenticatePusher`,
-        });
-
+        PusherHelper.setup();
```

Comment:
Didn't end up needing this `PusherHelper` change, but I do think it's a potentially useful abstraction, so I left it here. I can remove it if you prefer